views/pages/get-involved/clubs.pug

```diff
@@ -0,0 +1,16 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+    link(href="/assets/css/pages/get-involved/clubs.css" rel="stylesheet")
+
+block body
+    +banner-image("/assets/img/banners/mathsoc-wall.jpeg")
+    div(class="content")
+    section(class="white")

```

Comment:
could you add a header to this page? similar to what you did for the community page

Reply:
Done - header is "Clubs"